backend/routes/usernameRoute.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+
+router.post('/check-username',  async (req, res) => {
+    const {username} = req.body;
+    const findUser = await prisma.user.findUnique({
+        where: {username}
+    })
+
+    const exists = findUser !== null
+    res.json({exists});
+})
+
+router.put('/', async (req, res) => {
```

Comment:
We don't need an extra api endpoint here. We can include this "success" logic in the above route you defined above

Reply:
which api endpoint are you talking about? i think i need router.put no?